src/resmom_win/mom_stage_file.c

```diff
@@ -293,16 +316,19 @@ main(int argc, char *argv[])
 		rmjobdir(rqcpf->rq_jobid, pbs_jobdir, NULL, NULL);
 	}
 
-	/* log the number of files/directories copied and the time it took */
-	copy_stop = copy_stop - copy_start;
-	sprintf(log_buffer, "staged %d items %s over %d:%02d:%02d",
-		num_copies, (dir == STAGE_DIR_OUT) ? "out" : "in",
-		copy_stop/3600, (copy_stop%3600)/60, copy_stop%60);
-	log_event(PBSEVENT_DEBUG2, PBS_EVENTCLASS_JOB, LOG_DEBUG,
-		rqcpf->rq_jobid, log_buffer);
+	/* if operation is successful, log the number of files/directories copied and the time it took */
+	if (!err) {
```

Comment:
I think err variable is not required. We can directly use the 'rc' variable to check if there is any error.